modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -282,6 +289,20 @@ For example, selecting two streams from the same module is
 invalid.
 ====
 
+.Procedure: Disabling Modularity
+. In the {productname} {webui}, navigate to your {rhel}{nbsp}8 CLM project.
```

Comment:
Should we better use RHEL 9 as an example here?
I do not know if the use of the none matcher in RHEL 8 result in a bootable system